Ginger/Ginger/AutomatePageLib/NewAutomatePage.xaml.cs

```diff
@@ -126,16 +126,19 @@ private void SetRunSetDBLite(BusinessFlow businessFlowToLoad)
             {
                 if (mRunSetReport == null)
                 {
-                    mRunSetReport = new RunSetReport();
+                    mRunSetReport = new RunSetReport();                    
                     mRunSetReport.SetDataForAutomateTab();
-                    if (!ClearPreviousAutoRunSetDocumentLiteDB())
+                    ExecutionLoggerManager.RunSetReport = mRunSetReport;
+                    bool isAutoRunSetExists = SetOrClearPreviousAutoRunSetDocumentLiteDB(false);
+                    if (!isAutoRunSetExists)
                     {
                         mRunner.ExecutionLoggerManager.BusinessFlowEnd(0, businessFlowToLoad, true);
                         mRunner.ExecutionLoggerManager.RunnerRunEnd(0, mRunner, offlineMode:true);
                         mRunner.ExecutionLoggerManager.mExecutionLogger.SetReportRunSet(mRunSetReport, "");
+                        SetOrClearPreviousAutoRunSetDocumentLiteDB(false);
                         return;
                     }
-                    ExecutionLoggerManager.RunSetReport = mRunSetReport;
+                    SetOrClearPreviousAutoRunSetDocumentLiteDB(true);                  
```

Comment:
Codacy found an issue: [Remove this default value assigned to parameter 'isClear'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6406925)